courseraresearchexports/models/ExportRequest.py

```diff
@@ -0,0 +1,371 @@
+            id=json_request.get('id'),
+            status=json_request.get('status'),
+            download_link=json_request.get('downloadLink'),
+            metadata=ExportRequestMetadata.from_json(
+                json_request.get('metadata')))
+
+    @property
+    def created_at(self):
+        if self.metadata and self.metadata.created_at:
+            return self.metadata.created_at
+        else:
+            return datetime.fromtimestamp(0)
+
+    def download(self, dest):
+        """
+        Download an export archive associated with this export request
+        :param dest:
+        :return output_filename:
+        """
+        if self.export_type == EXPORT_TYPE_EVENTING:
+            logging.error(
+                'Generate download links to access eventing export requests'
+                'using `courseraresearchexports jobs eventing_download_links`.'
+                ' Please refer to the Coursera Export Guide for details'
+                'https://coursera.gitbooks.io/data-exports/content/'
+                'introduction/programmatic_access.html')
+            raise ValueError(
+                'Require export_type = {}'.format(EXPORT_TYPE_SCHEMAS))
+        elif not self.download_link:
+            if self.status in ['PENDING', 'IN_PROGRESS']:
+                logging.error(
+                    'Export request {} is currently {} and is not ready for'
+                    'download. Please wait until the request is completed.'
+                    .format(self.id, self.status))
+                raise ValueError(
+                    'Export request is not yet ready for download')
+            elif self.status == 'TERMINATED':
+                logging.error(
+                    'Export request has been TERMINATED. Please contact '
+                    'data-support@coursera.org if we have not resolved this '
+                    'within 24 hours.')
+                raise ValueError('Export request has been TERMINATED')
+            else:
+                logging.error('Download link was not found.')
+                raise ValueError('Download link was not found')
+
+        if not os.path.exists(dest):
+            logging.debug('Creating destination folder: {}'.format(dest))
+            os.makedirs(dest)
+
+        export_filename = urlparse(self.download_link).path.split('/')[-1]
+        response = requests.get(self.download_link, stream=True)
+        chunk_size = 1024 * 1024
+        output_filename = os.path.join(dest, export_filename)
+        logging.debug('Writing to file: {}'.format(output_filename))
+
+        with open(output_filename, 'wb') as f:
+            for data in tqdm(
+                    iterable=response.iter_content(chunk_size),
+                    total=int(response.headers['Content-length']) / chunk_size,
+                    unit='MB',
+                    desc=export_filename):
+                f.write(data)
+
+        return output_filename
+
+
+def datetime_to_ts(dt):
```

Comment:
`ts` -> `unix_ms` (since timestamp isn't very informative of what the format is)